src/main/resources/cloneRepoAndPdd.sh

```diff
@@ -1,5 +1,5 @@
-whoami
-pwd
-mkdir self-todos-tmp-%s && cd self-todos-tmp-%s
-git clone git@%s:%s repo && cd repo
-java -jar /usr/local/bin/todo-finder-cli.jar
+whoami;
+pwd;
+mkdir self-todos-tmp-%s && cd self-todos-tmp-%s;
+git clone git@%s.com:%s;
```

Comment:
@criske comanda asta nu e ok. Asa era initial si am schimbat-o... poate trebuia sa scriu niste documentatie. Exemplu asta e cu Github, dat la fel e si pt GitLab:

``git clone git@github.com:repo/test `` va clona prin SSH acel repository, dar va folosi tot timpul aceeasi cheie SSH care e pe server. Asta inseamna ca nu vom putea clona decat folosind un singur PM.

Remember, putem avea oricati PM-i si fiecare trebuie sa-si foloseasca cheia lui SSH pt a clona prin contul lui. Un PM poate trebuie sa cloneze un repo privat la care doar el are acces, deci clar nu putem folosi aceeasi cheie pt toti.

In schimb:

``git clone git@github-charlesmike:test/repo`` va merge deoarece am facut o setare pe server (in clientul de SSH) care zice ca ``github-charlesmike`` e un alias catre githuh.com, folosind cheia lui charlesmike.

Reply:
@criske aici sunt setarile de care ziceam, alias pt fiecare PM:https://gist.github.com/oanhnn/80a89405ab9023894df7

Reply:
@amihaiemil e nevoie de pipeline-ul ala ca sa pot testa webhook-urile direct prin localhost. (si oricum github/github nu te lasa sa ai callbacks prin locala). Asa ca pipedream sta intre mine si github/gitab.

> @criske comanda asta nu e ok. Asa era initial si am schimbat-o... poate trebuia sa scriu niste documentatie. Exemplu asta e cu Github, dat la fel e si pt GitLab:
> 
> `git clone git@github.com:repo/test ` va clona prin SSH acel repository, dar va folosi tot timpul aceeasi cheie SSH care e pe server. Asta inseamna ca nu vom putea clona decat folosind un singur PM.
> 
> Remember, putem avea oricati PM-i si fiecare trebuie sa-si foloseasca cheia lui SSH pt a clona prin contul lui. Un PM poate trebuie sa cloneze un repo privat la care doar el are acces, deci clar nu putem folosi aceeasi cheie pt toti.
> 
> In schimb:
> 
> `git clone git@github-charlesmike:test/repo` va merge deoarece am facut o setare pe server (in clientul de SSH) care zice ca `github-charlesmike` e un alias catre github.com, folosind cheia lui charlesmike.

Mersi de clarificari. Acum imi amintesc ca am discutat la un moment dat de chestia asta. (eu propuneam parca sa trimitem tokenul managerului prin https ceva de genu `git clone https://<manager>:<token>@github.com/<owner>/<name>.git`)`
Am sa fac modificarile alea pe ssh config sa fiu la zi cu scriptul si o sa incerc din nou pt gitlab. :D